doc/api/errors.md

```diff
@@ -191,16 +191,19 @@ provide a text description of the error.
 All errors generated by Node.js, including all system and JavaScript errors,
 will either be instances of, or inherit from, the `Error` class.
 
-### `new Error(message)`
+### `new Error(message[, options])`
 
 * `message` {string}
+* `options` {Object}
+  * `cause` {any} The error that caused the newly created error.
 
 Creates a new `Error` object and sets the `error.message` property to the
 provided text message. If an object is passed as `message`, the text message
-is generated by calling `message.toString()`. The `error.stack` property will
-represent the point in the code at which `new Error()` was called. Stack traces
-are dependent on [V8's stack trace API][]. Stack traces extend only to either
-(a) the beginning of _synchronous code execution_, or (b) the number of frames
+is generated by calling `message.toString()`. The `error.cause` property is set
+to the optionally provided `cause`. The `error.stack` property will represent
```

Comment:
I can see that this wasn't brought by this PR, but I think it's worth fixing: we're are calling `.toString`.

```js
console.log(
  new Error({
    toString() {
      return "toString";
    },
    [Symbol.toPrimitive]() {
      return "@@toPrimitive";
    },
  }).message
); // @@toPrimitive
```

```suggestion
is generated by calling `String(message)`. If a `cause` is provided, it is set
as `error.cause`. The `error.stack` property will represent
```